app/assets/javascripts/editor.js

```diff
@@ -105,16 +105,15 @@ $E = {
   generate_preview: function(id,text) {
     $('#'+id)[0].innerHTML = marked(text)
   },
-  toggle_preview: function(id) {
+  toggle_preview: function() {
     $E.preview[0].innerHTML = marked($E.textarea.val());
```

Comment:
`$E.preview[0]` this is probably targetting 1st comment, it works in case of Notes as Notes have just 1 comment form at the end, but in case of Question, there would be comment form with each answer.